### DISCUSSION TITLE
[Script request] LabCA

### DISCUSSION THREAD

**Discussion by jwgn** [community-scripts/ProxmoxVE#214] [Request script]
### Application Name

LabCA

### Website

https://lab-ca.net/

### Description

A private Certificate Authority for internal (lab) use, based on the open source ACME Automated Certificate Management Environment implementation from Let’s Encrypt (tm).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by jwgn** [comment]
This is labeled  `in project pipeline`, but where is this pipeline? I have searched [here](https://github.com/orgs/community-scripts/projects/9) but could not find anything related to LabCA.

**Reply by oOStroudyOo** [reply]
Backlog column, right at the bottom.
https://github.com/orgs/community-scripts/projects/9?pane=issue&itemId=102338511
